src/getting-started.md

```diff
@@ -131,204 +125,189 @@ are two major downsides:
-limited to an individual deck. This means you can use many different
-types of notes in a particular deck, or have different cards generated
-from a particular note in different decks. When you add notes using the
+limited to an individual deck. This means you can use different
+notes types in a single deck, or have cards generated from the
+ same note in different decks. When you add notes using the
 Add window, you can select what note type to use and what deck to use,
 and these choices are completely independent of each other. You can also
-change the note type of some notes [after you’ve already created them](browsing.md).
+change the note type of some notes [after you've already created them](browsing.md).
 
 ### Collection
 
-Your 'collection' is all the material stored in Anki – your cards,
+Your _collection_ is all the material stored in Anki: your cards,
 notes, decks, note types, deck options, and so on.
 
 ## Shared Decks
 
-You can watch [a video about Shared Decks and Review
+You can watch a video about [Shared Decks and Review
 Basics](http://www.youtube.com/watch?v=QS2G-k2hQyg&yt:cc=on) on YouTube.
 
 The easiest way to get started with Anki is to download a deck of cards
-someone has shared:
+someone else has shared:
 
-1. Click the “Get Shared” button at the bottom of the deck list.
+1. Click the "Get Shared" button at the bottom of the deck list.
 
-2. When you’ve found a deck you’re interested in, click the “Download”
+2. When you've found a deck you're interested in, click the "Download"
    button to download a deck package.
 
-3. Double-click on the downloaded package to load it into Anki, or
-   File→Import it.
+3. Double-click the downloaded package to import it into Anki, or go to
+   File → Import.
 
-Please note that it’s not currently possible to add shared decks
-directly to your AnkiWeb account. You need to import them with the
-desktop program, then synchronize to upload them to AnkiWeb.
+Note: It’s not currently possible to add shared decks
+directly to your AnkiWeb account. Please first import them to the
+desktop app, then synchronize to upload the decks to AnkiWeb.
 
 Creating your own deck is the most effective way to learn a complex
 subject. Subjects like languages and the sciences can’t be understood
-simply by memorizing facts — they require explanation and context to
-learn effectively. Furthermore, inputting the information yourself
+simply by memorizing facts — you need explanation and context to
+learn them effectively. Furthermore, inputting the information yourself
 forces you to decide what the key points are, leading to a better
 understanding.
 
 If you are a language learner, you may be tempted to download a long
 list of words and their translations, but this won’t teach you a
 language any more than memorizing scientific equations will teach you
-astrophysics. To learn properly, you need textbooks, teachers, or
+astrophysics. To learn properly, we believe you need textbooks, teachers, or
```

Comment:
```suggestion
astrophysics. To learn properly, you also need textbooks, teachers, or
```
Using "we believe" undercuts the statement. If it's not true and accurate, it's not worth saying.

Reply:
It's important to emphasise that opinions are subjective especially because people are divided on "how best to learn". For example, I personally don't think you need textbook/teachers, real world sentences are important but my beliefs are more nuanced.

Reply:
```suggestion
astrophysics. To learn properly, you might need textbooks, teachers, or
```